docs/user/dashboard/create-visualizations.asciidoc

```diff
@@ -240,6 +243,29 @@ image::images/esql-visualization-customization-options.png[Appearance customizat
 
 . Return to the previous menu, then **Apply and close** the configuration to save your changes.
 
+[[create-alert-from-esql-viz]]
+[float]
+== Create an alert from your {esql} visualization
+
+Once you've created an {esql} panel, you can create an {es} threshold rule directly from the visualization panel based on the data it displays. When you do this, the rule query is automatically generated and either describes the data and sets a specific threshold, or describes the data without setting a specific threshold.
+
+[NOTE]
+====
+{elastic-sec} rule types are not supported.
+====
+
+To create a rule with the threshold pre-specified:
+
+* Right-click a data point in the visualization and click *Add alert rule*. This opens the *Create rule* flyout. The generated query will define a threshold that corresponds to the data point you selected.
+* {observability-guide}/create-an-elasticsearch-query-rule.html[Configure] your {es} rule.
```

Comment:
Hey! This might be the page that you're looking for: https://www.elastic.co/guide/en/kibana/8.18/rule-type-es-query.html

```suggestion
* {kib-ref}/rule-type-es-query.html[Configure] your {es} rule.
```